.travis.yml

```diff
@@ -1,43 +1,42 @@
 language: php
-
 sudo: false
+dist: trusty
+
+services:
+  - docker
+
+php:
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
-
-php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2]
-
-env:
-  global:
-    - WEBDRIVER=selenium
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.0
-      env: WEBDRIVER=selenium-remote
-      sudo: required
-      services:
-        - docker
-    - php: 5.3
-      dist: precise
-      # Force using PHP 5.6 for the test server as PHP 5.3 does not have the builtin webserver
-      env: MINK_PHP_BIN=~/.phpenv/versions/5.6/bin/php
+    - $HOME/.composer/cache
 
-before_script:
-  - sh bin/run-"$WEBDRIVER".sh
-
-  - composer install
+install:
+  - travis_retry composer self-update
+  - travis_retry composer update --no-interaction $DEPENDENCIES
 
-  # Start a webserver for web fixtures.
-  - vendor/bin/mink-test-server > /dev/null 2>&1 &
+before_script:
+  - mkdir ./logs
+  - docker run -p 4444:4444 --network=host -v /dev/shm:/dev/shm --shm-size 2g selenium/standalone-firefox:2.53.1 &> ./logs/selenium.log &
+  - until $(echo | nc localhost 4444); do sleep 1; echo Waiting for Selenium server on port 4444...; done; echo "Selenium server started"
+  # https://bugs.php.net/bug.php?id=71443
+  - USE_ZEND_ALLOC=0 travis_retry php -d memory_limit=8G -S localhost:8002 -t ./vendor/mink/driver-testsuite/web-fixtures &> ./logs/mink-test-server.log &
+  - until $(echo | nc localhost 8002); do sleep 1; echo waiting for PHP server on port 8002...; done; echo "PHP server started"
 
-script: phpunit -v --coverage-clover=coverage.clover
+script:
+  - phpunit -v --coverage-clover=coverage.clover
 
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
 
 after_failure:
-  - cat /tmp/webdriver_output.txt
+#  - cat ./logs/selenium.log
```

Comment:
Any reason for changing what is displayed in case of an error (before WebDriver output was displayed, but now you're displaying web server output)?

Reply:
- Selenium produces quite a lot of logs and Travis CI has limit 4Mb
- Server logs were added because it had "Segmentation fault" on PHP 7.1, but thanks to `USE_ZEND_ALLOC` issue seems to be gone